L1Trigger/L1THGCal/interface/be_algorithms/HGCalHardwareInterface.h

```diff
@@ -0,0 +1,71 @@
+#ifndef __L1Trigger_L1THGCal_HGCalHardwareInterface_h__
+#define __L1Trigger_L1THGCal_HGCalHardwareInterface_h__
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/L1THGCal/interface/HGCalCluster.h"
+#include "DataFormats/L1THGCal/interface/HGCalMulticluster.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
```

Comment:
I don't see any log nor parameter sets used there, so the 2 corresponding `include` could be removed